src/combinatorics/simple_graph/basic.lean

```diff
@@ -360,6 +363,11 @@ The maximum degree of all vertices
 def max_degree (G : simple_graph V) [nonempty V] [decidable_rel G.adj] : ℕ :=
 finset.max' (univ.image (λ (v : V), G.degree v)) (nonempty.image univ_nonempty _)
 
+/-! The following lemmas about `fintype.card` use noncomputable decidable instances to get fintype
+assumptions. -/
+section
+open_locale classical
```

Comment:
I think this will cause issues when people will try to do `card` when they have the correct `decidable` on the graph carrier or adjacency relation. But getting that right is probably outside the scope of this PR.